docs/about/model.md

```diff
@@ -25,28 +25,28 @@ Multiple coordinators can coexist in the network running Hermez nodes.
 
-All bids will be processed by the auction smart contract , and all the HEZ tokens placed will be used as follows:
+All bids will be processed by the auction smart contract , and all of the HEZ tokens placed will be used as follows:
 
 - 30% will be burned (permanently removed).
 - 40% will be automatically and permanently transferred to a donations account controlled by the Ethereum Foundation. These donations will be initially sent to Gitcoin quadratic funding grants but with the future-proof ability to donate also to other quadratic funding matching pools as they become available;
 - 30% will be allocated as Hermez Network usage incentives, compensating active engagement and network adoption, e.g. rewarding transaction and rewarding the holding of specific tokens in Hermez L2 addresses, instead of on L1 Ethereum addresses.
 
 
-The auction process incentivise efficiency, as coordinators need to include as many transactions as possible in each time slot in order to compensate for their bidding costs and operative expenses.
+The auction process incentivises efficiency, as coordinators need to include as many transactions as possible in each time slot to compensate for their bidding costs and operative expenses.
 
 To prevent bidders from buying up all the slots in one go, nobody will be able to bid on a specific slot more than one month in advance. And the auction will be closed two time slots before the time of creating the slot.
 
 The auction will be structured in a series of six time slots to cover one hour (%0-5), with 10 HEZ as the initial minimal bidding price for all of them.
 
-The first bid in each time slot must be over the minimal bidding price in order to be accepted as valid. Thereafter, any bid placed in the auction should outbid the previous bid by at least 10%.
+The first bid in each time slot must be over the minimum bidding price in order to be accepted as valid. Thereafter, any bid placed in the auction should outbid the previous bid by at least 10%.
 
 
-The minimum bidding amount for each slot in an auction series will be decided by the network governance (see section "Hermez governance"), and it will be possible to change it dynamically, affecting future, even already open, auctions.
+The minimum bidding amount for each slot in an auction series will be decided by the network governance (see section "Hermez Governance"), and it will be possible to change it dynamically, affecting future, even already open, auctions.
 
-Also, governance will be able to implement the effective decentralization of the network by locking the price of specific slots of the auction to 0 HEZ and so don't requiring any minimal bid, being this an irreversible configuration.
+Also, governance will be able to implement the effective decentralisation of the network by locking the price of specific slots of the auction to 0 HEZ and therefore wouldn't require any minimal bid, this being an irreversible configuration.
 
 ## Users
 
```

Comment:
remove line

Reply:
Jenna, i you render the document, there is a big space between the first and second paragraph of the Users section. You need to delete one line so that it looks better. thanks